Components/Card/card.css

```diff
@@ -0,0 +1,335 @@
+  border: 1px solid var(--color-gray);
+  border-radius: 50%;
+  font-size: 1rem;
+  font-weight: 600;
+}
+
+.saiyan-horizontal-card .card__CTA__container {
+  margin-top: 2rem;
+  display: flex;
+  flex-direction: column;
+  gap: 1rem;
+}
+
+.saiyan-horizontal-card .card__CTA__btn {
+  padding: 0.5rem 1rem;
+  /* border: 1px solid var(--horizontal-card-CTA-border-color); */
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  border-radius: 0.6rem;
+  text-decoration: none;
+  font-weight: 600;
+  background-color: var(--card-primary-color);
+  color: var(--color-white);
+}
+
+.saiyan-horizontal-card .card__CTA__btn--secondary {
+  background-color: transparent;
+  color: var(--card-primary-color);
+  border: 1px solid var(--card-primary-color);
+}
+
+.saiyan-horizontal-card .card__price__stricked {
+  color: var(--horizontal-card-price-stricked-color);
+  text-decoration: line-through;
+  font-size: 1.5rem;
+}
+
+.saiyan-horizontal-card .card__price__discount {
+  color: var(--horizontal-card-price-discount);
+}
+
+/* horizontal card end */
+
+/* vertical */
+
+.saiyan-vertical-card {
+  width: max-content;
+  height: fit-content;
+  border-radius: 0.6rem;
+  box-shadow: var(--vertical-card-shadow);
+  position: relative;
+}
+
+.saiyan-vertical-card .card__img {
+  width: 220px;
+  height: 280px;
+  border-radius: 0.6rem;
+}
+
+.saiyan-vertical-card .card__heading {
+  color: var(--color-black);
+  font-size: 1rem;
+  font-weight: 600;
+  margin-bottom: 0.4rem;
+  margin-top: 0;
+}
+
+.saiyan-vertical-card .card__subtext {
+  color: var(--color-black);
+  font-size: 1rem;
+  font-weight: 400;
+  margin-top: 0;
+  margin-bottom: 0;
+}
+
+.saiyan-vertical-card .card__body {
+  padding: 1rem;
+}
+
+.saiyan-vertical-card .card__price__container {
+  margin: 0.4rem 0;
+}
+
+.saiyan-vertical-card .card__price {
+  color: var(--color-black);
+  font-size: 0.9rem;
+  font-weight: 700;
+}
+
+.saiyan-vertical-card .card__price__stricked {
+  font-size: 0.8rem;
+  text-decoration: line-through;
+  font-weight: 500;
+}
+
+.saiyan-vertical-card .card__price__discount {
+  color: var(--card-discount-color);
+  font-weight: 500;
+  font-size: 0.9rem;
+}
+
+.saiyan-vertical-card .card__actions {
+  margin: 1.5rem 0;
+  margin-bottom: 0.5rem;
+}
+
+.saiyan-vertical-card .card__btn {
+  background-color: var(--vertical-card-primary-color);
+  color: var(--color-white);
+  text-decoration: none;
+  padding: 0.6rem 1rem;
+  border-radius: 0.6rem;
+  font-weight: 600;
+}
+
+.saiyan-vertical-card .card__close__btn__container {
+  position: absolute;
+  top: 1rem;
+  right: 1rem;
+}
+
+.saiyan-vertical-card .card__close__btn {
+  font-weight: 700;
+  font-size: 1.5rem;
+}
+
+.saiyan-vertical-card .card__btn {
+  background-color: var(--card-primary-color);
+  color: var(--color-white);
+  text-decoration: none;
+  padding: 0.6rem 1rem;
+  border-radius: 0.6rem;
+  font-weight: 600;
+}
+
+.saiyan-vertical-card .card__btn__secondary {
```

Comment:
Suggestion: `saiyan-vertical-card` you can try to think of changing the class-name.

Reply:
want to change it but already using it in ecommerce, and in short of time, will surely change it to something short if i get time.